test/custom-production.js

```diff
@@ -1,7 +1,40 @@
-"use strict";
-
 import expect from "expect";
+
 import { parse, write } from "webidl2";
+import { Base } from "../lib/productions/base.js";
+import {
+  autoParenter,
+  type_with_extended_attributes,
+} from "../lib/productions/helpers.js";
+
+class CustomAttribute extends Base {
```

Comment:
Good, can you replace the existing test below to use this? I think this one is better than `const customProduction` in it.

Reply:
Addressed in [9cfc34c](https://github.com/w3c/webidl2.js/pull/731/commits/9cfc34c560c39960f13eb06b16c183e01bfd2165).